56/metallrooch/music_duration.py

```diff
@@ -0,0 +1,55 @@
+"""
+Writes track's name and length into result.csv 
+"""
+
+import subprocess
+import re
+import datetime
+import argparse
+import csv
+
+from pathlib import Path, PurePath, PurePosixPath
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('dir_path', type=str, help='Path to dir')
+    args = parser.parse_args()
+
+    with open('result.csv', 'w', newline='') as csvfile:
+        fieldnames = ['track_name', 'length']
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+
+        writer.writeheader()
+
+        p = Path(args.dir_path)
+        t0 = datetime.timedelta(seconds=0)
+
+        for i in p.glob('*.*'):
```

Comment:
I would try to use more descriptive variable names than `p`, `t0` and `i`
Need to look into `pathlib`, nice!